src/desktop/apps/consign/components/create_account/index.js

```diff
@@ -5,14 +5,13 @@ import React from 'react'
 import _SignUp from '../sign_up'
 import block from 'bem-cn-lite'
 import { connect } from 'react-redux'
-import { incrementStep } from '../../client/actions'
```

Comment:
@sweir27 - Odd that babel didn't pick this up before but it looks like this obj doesn't exist (doesn't seem to be used).